modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItInternalTableScanTest.java

```diff
@@ -349,18 +349,18 @@ public void testNullPointerExceptionIsThrownInCaseOfNullSubscription() {
      * @param entryKey Key.
      * @param entryVal Value
      * @return {@link BinaryRow} based on given key and value.
-     * @throws java.io.IOException If failed to close output stream that was used to convertation.
      */
-    private static @NotNull BinaryRow prepareRow(@NotNull String entryKey,
-            @NotNull String entryVal) throws IOException {
-        byte[] keyBytes = ByteUtils.toBytes(entryKey);
-
-        try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
-            outputStream.write(keyBytes);
-            outputStream.write(ByteUtils.toBytes(entryVal));
-
-            return new ByteBufferRow(keyBytes);
-        }
+    private static BinaryRow prepareRow(String entryKey, String entryVal) {
+        return new RowAssembler(
+                new SchemaDescriptor(
+                        1,
+                        new Column[]{new Column("key", NativeTypes.stringOf(100), false)},
+                        new Column[]{new Column("val", NativeTypes.stringOf(100), false)}
+                ), 1, 1
+        )
+                .appendString(Objects.requireNonNull(entryKey, "entryKey"))
+                .appendString(Objects.requireNonNull(entryVal, "entryVal"))
+                .build();
```

Comment:
Let's move schema to upper level.
```suggestion
    private static BinaryRow prepareRow(SchemaDescriptor desc, Object... values) {
        assert values.length == desc.length();
        RowAssembler asm = new RowAssembler(desc , 1, 1);
        
        for (int i=0; i<values.length; i++){
                assert values[i] != null || desc.column(i).nullable();
                RowAssembler.writeValue(asm, desc.column(i), values[i]);
       }
       return asm.build();
```

Reply:
moved to constant